cmd/port-layer-server/main.go

```diff
@@ -76,8 +79,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	// load the vch config
+	src, err := extraconfig.GuestInfoSource()
+	if err != nil {
+		log.Errorf("Unable to load configuration from guestinfo")
+		return
+	}
+
+	extraconfig.Decode(src, &vchConfig)
+
 	logcfg := viclog.NewLoggingConfig()
-	if ploptions.PortLayerOptions.Debug {
+	if vchConfig.Diagnostics.DebugLevel {
```

Comment:
I think issue #595 is too old, just occasionally not closed. Portlayer already query guestinfo configuration in Init method: https://github.com/vmware/vic/blob/master/lib/portlayer/portlayer.go#L60
Might consolidate these two queries